app/models/ect_at_school_period.rb

```diff
@@ -5,13 +5,13 @@ class ECTAtSchoolPeriod < ApplicationRecord
   enum :appropriate_body_type,
        { teaching_induction_panel: "teaching_induction_panel",
          teaching_school_hub: "teaching_school_hub" },
-       validate: { allow_nil: true },
+       validate: { message: "Must be teaching_induction_panel or teaching_school_hub" },
        suffix: :ab_type
 
   enum :programme_type,
        { provider_led: "provider_led",
          school_led: "school_led" },
-       validate: { allow_nil: true },
+       validate: { message: "Must be provider_led or school_led" },
```

Comment:
Shouldn't be underscored I think "provider-led" and "school-led" in messages

Reply:
Same as above.